spec/unit/peeps_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+require 'peeps'
+
+describe Peep do
+
+  describe '#all' do
+    it 'lists all the peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
+
+      peep1 = Peep.create(message: 'This is my first peep!')
+      peep2 = Peep.create(message: 'This is my second peep!')
+      peep3 = Peep.create(message: 'This is my third peep!')
+
+      peeps_list = Peep.all
+
+      expect(peep1.message).to include "This is my first peep!"
+      expect(peep2.message).to include "This is my second peep!"
+      expect(peep3.message).to include "This is my third peep!"
+
```

Comment:
Here, you're testing `create` rather than `all` because you don't use `peeps_list` in your assertion.